shared/test_artifacts/generator.go

```diff
@@ -34,28 +40,31 @@ var (
 
 // TODO (Team) this is meant to be a **temporary** replacement for the recently deprecated
 // 'genesis config' option. We need to implement a real suite soon!
-func NewGenesisState(numValidators, numServiceNodes, numApplications, numFisherman int) (genesisState modules.GenesisState, validatorPrivateKeys []string) {
-	apps, appsPrivateKeys := NewActors(MockActorType_App, numApplications)
-	vals, validatorPrivateKeys := NewActors(MockActorType_Val, numValidators)
-	serviceNodes, snPrivateKeys := NewActors(MockActorType_Node, numServiceNodes)
-	fish, fishPrivateKeys := NewActors(MockActorType_Fish, numFisherman)
-	return modules.GenesisState{
-		ConsensusGenesisState: &MockConsensusGenesisState{
+func NewGenesisState(numValidators, numServiceNodes, numApplications, numFisherman int) (modules.GenesisState, []string) {
+	apps, appsPrivateKeys := NewActors(types.ActorType_App, numApplications)
+	vals, validatorPrivateKeys := NewActors(types.ActorType_Validator, numValidators)
+	serviceNodes, snPrivateKeys := NewActors(types.ActorType_ServiceNode, numServiceNodes)
+	fish, fishPrivateKeys := NewActors(types.ActorType_Fisherman, numFisherman)
+
+	genesisState := runtime.NewGenesis(
```

Comment:
Non-Blocker:

Is there no way to use `runtime` for this?

Reply:
If you mean to move test_artifacts under runtime. I have done it.